cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
> This has nothing to do with merging scripts

If we abolish flock, then we don't need to turn trivial rube-goldberg solutions into 3, 4 or 5 script solutions.

> It is more complicated.

That is matter of opinion.

> Usage is limited to one instance per process name.

Correct.  This is nice when we're looking for singleton.

> OTOH, you are not limited to pass a command to `flock`

Nor any other singleton approach.  The limitations are dependent on how we match the process.  Seeing a script look for itself already running is more complicated to troubleshoot, but watching a script call flock, which calls a wrapper around a script which calls a script is also complex.